vite/public/edge-surf-game-v1/resources/js/parse_html_subset.js

```
@@ -0,0 +1,247 @@
+
+    if (window.trustedTypes) {
+      s = toTrustedHtml(s);
+    }
+
+    // This does not execute any scripts because the document has no view.
+    const df = r.createContextualFragment(s);
+    walk(df, function(node) {
+      switch (node.nodeType) {
+        case Node.ELEMENT_NODE:
+          assertElement(tags, node);
+          const nodeAttrs = node.attributes;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)